app/views/cases/offender_sar/_requested_info_step.html.slim

```diff
@@ -8,8 +8,7 @@
 
 = form_for @case, url: case_create_action(@case.object), as: :offender_sar do |f|
 
-  = f.text_area :message, {rows: 12,
-    form_hint_text: t('helpers.hint.offender_sar_case_form.message')}
+  = f.text_area :message, {rows: 12}
```

Comment:
Leave spaces between curly brace and contents i.e. `{ rows: 12 }`

Reply:
Yes I like that too, done. thanks